src/components/Calendar/SingleDayPicker.story.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import withTests from '../../util/withTests';
+import { SingleDayPicker } from '.';
+
+class CalendarStoryPicker extends Component {
+  state = { date: null, focused: null };
```

Comment:
Another potential use case for `State`.

Reply:
Switched to `recompose`.

Reply:
But did you guys check whether using the class component fixes the problem where our automatic docs doesn't work because it doesn't see the root component as the main component?

Or does it treat the class component as the root component and tries to document that which obviously is also pointless?

Reply:
Not really applicable in this case and ... probably not :)